docs/user-guide/glossary.md

```diff
@@ -188,8 +188,8 @@ microgrid are connected with each other.  Some of the ways in which the SDK uses
 the component graph are:
 
   - figure out how to calculate high level metrics like
-[`grid_power`][frequenz.sdk.timeseries.grid.Grid.power],
-[`consumer_power`][frequenz.sdk.timeseries.consumer.Consumer.power],
+[`grid.power`][frequenz.sdk.timeseries.grid.Grid.power],
+[`consumer.power`][frequenz.sdk.timeseries.consumer.Consumer.power],
```

Comment:
Shouldn't it be like this?  That's how it is represented elsewhere: https://github.com/frequenz-floss/frequenz-sdk-python/blob/4fb5a8802fe5f26206c6408de1389e7de9246bca/src/frequenz/sdk/microgrid/__init__.py#L80

```suggestion
[`grid().power`][frequenz.sdk.timeseries.grid.Grid.power],
[`consumer().power`][frequenz.sdk.timeseries.consumer.Consumer.power],
```